soweego/wikidata/api_requests.py

```diff
@@ -31,9 +33,63 @@
 LOGGER = logging.getLogger(__name__)
 
 WIKIDATA_API_URL = 'https://www.wikidata.org/w/api.php'
+# Good ol' blacklist:
+# https://www.wikidata.org/wiki/Wikidata:Primary_sources_tool/URL_blacklist
+URL_BLACKLIST_PAGE = 'Wikidata:Primary_sources_tool/URL_blacklist'
 BUCKET_SIZE = 500
 
 
+def get_url_blacklist() -> set:
+    """Retrieve a blacklist with URL domains of low-quality sources
+
+    :return: the set of blacklisted domains
+    """
+    params = {
+        'action': 'parse',
+        'format': 'json',
+        'prop': 'text',
+        'page': URL_BLACKLIST_PAGE
+    }
+    response_body = _make_request(params)
+
+    # Failed API request
+    if response_body is None:
+        return None
+
+    # Paranoid checks over the response JSON keys
```

Comment:
can be simplified with a try..except:

```
try:
    star = response_body['parse']['text']['*']
except KeyError:
    LOGGER.error(...)
    return None
```

Reply:
Fair enough, we can lose fine-grained logging for the sake of code simplicity.